src/main/resources/db/migration/V1_0__create_initial_referral.sql

```diff
@@ -0,0 +1 @@
+create table referral (id uuid not null, created timestamp, primary key (id));
```

Comment:
What do you think about auto-assigning a default primary key with `uuid_generate_v4()`, rather than relying on the application code to do that?

Reply:
i think i'd prefer to let hibernate do as much of this as possible, e.g. also auto-created timestamps etc, rather than by hand in these files.